src/material-experimental/theming/_format-tokens.scss

```diff
@@ -0,0 +1,5 @@
+// DO NOT REMOVE
+// This function is used internally.
+@function format($systems) {
```

Comment:
Should we change the name to indicate that this is a private API?

Reply:
Done. Let me know if you would like a different name